README.md

```diff
@@ -863,11 +865,11 @@ Except for the `unitTests`,  CMake needs to find a python3 interpreter in the sy
 
 | Name               | Language  | Location    | Command<br>_(in build directory)_ | CMake Option to Build          |
 |:--                 |:--        |:--                      |:--                    |:--                             |
-| bashTests          | python    | tests/bash\_tests       | $ ctest -R bash       | -DEXIV2\_BUILD\_SAMPLES=ON     |
+| bashTests          | python    | tests/bash_tests       | $ ctest -R bash        | -DEXIV2_BUILD_SAMPLES=ON       |
 | bugfixTests        | python    | tests/bugfixes          | $ ctest -R bugfix     |                                |
 | lensTest           | C++       | tests/lens_tests        | $ ctest -R lens       |                                |
 | tiffTests          | python    | tests/tiff_test         | $ ctest -R tiff       |                                |
-| unitTests          | C++       | unitTests/              | $ ctest -R unit       | -DEXIV2\_BUILD\_UNIT\_TESTS=ON |
+| unitTests          | C++       | unitTests/              | $ ctest -R unit       | -DEXIV2_BUILD_UNIT_TESTS=ON    |
```

Comment:
I still think you want to escape all the underscores like `\_`, not remove all?

Reply:
I checked how it looks in Github and other Markdown viewer and it looks the same:

![image](https://user-images.githubusercontent.com/102645/153581578-7606e23c-0fa8-4532-a4a2-f0ecd6312b5c.png)

Therefore, I choose the option with less characters 😁

Reply:
Ok, it was just the preview here then, thanks.